### DISCUSSION TITLE
"Discovery of Super-enriched Gas ∼1 Gyr after the Big Bang"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#172] [Astrophysics and Physics]
"Discovery of Super-enriched Gas ∼1 Gyr after the Big Bang"
[https://iopscience.iop.org/article/10.3847/2041-8213/aceefe](https://iopscience.iop.org/article/10.3847/2041-8213/aceefe)
<<These high levels of enrichment contrast sharply with the N(H i)-weighted metallicity of other DLAs at comparable redshifts, and also with the available galactic chemical evolution models with and without Population III stars.>>

From AAS Nova: "Metal Before It Was Cool: Super-Enrichment in the Early Universe"
<<... the authors currently have no explanation for what could lead to such a substantial metal enrichment>>
[https://aasnova.org/2023/10/13/metal-before-it-was-cool-super-enrichment-in-the-early-universe/](https://aasnova.org/2023/10/13/metal-before-it-was-cool-super-enrichment-in-the-early-universe/)

**Comment by bgreenman** [comment]
The new discoveries of heavier elements in high-redshift galaxies prove that the universe is much older than 13.8 billion years old. it tales time for galaxies to develop.metal infused gas
